packages/base64/src/main.js

```diff
@@ -0,0 +1,124 @@
+/* eslint no-bitwise: ["off"] */
```

Comment:
For multi-module packages, our convention seems to be to have `main.js` just import and reexport from the other modules. Is it reasonable to do this even when a package starts as a single module, so it can grow into that more smoothly? Given the kind of `repackage.sh`-like script-based construction I'd hope for, I'd guess the answer is yes, because it enable one generic script to set up more structure.

Reply:
Also, to set up the structure parallel to our other properjs packages, I'd expect a separate `types.js` file.

Reply:
I’m hoping to only use a `types.js` file when we have some types to communicate among multiple modules. It’s certainly the pattern!

Reply:
ok. Resolving.